app/frontend/styles/publications/_status_totals.scss

```
@@ -0,0 +1,7 @@
+.flex-container {
+  display: flex;
+}
+
+.app-count-header {
+  color: govuk-colour("blue");
+}
```

```suggestion
.status-totals-component {
  &__row {
    display: flex;
  }

  &__count-header {
    color: govuk-colour("blue");
  }
}
```

We should scope these classes to the component. If they're not scoped to the component then they probably shouldn't be in this file.